tests/unittests/hrm2_tests.py

```
@@ -0,0 +1,1336 @@
+        dataset.reload()
+        self.assertIsNotNone(dataset.default_skeleton)
+        self.assertEqual(len(dataset.default_skeleton.labels), 25)
+        self.assertEqual(len(dataset.default_skeleton.edges), 24)
+        self.assertEqual(dataset.default_skeleton.labels[0], "nose")
+        self.assertEqual(dataset.default_skeleton.labels[8], "mid_hip")
+
+    @drop_datasets
+    def test_skeleton_with_keypoints(self):
+        """Test skeleton is compatible with HRM2 keypoint format."""
+        from fiftyone.utils.hrm2 import get_hrm2_skeleton, HRM2_JOINT_NAMES
+        import tempfile
```

Ruff (0.14.5)

Redefinition of unused `tempfile` from line 12

Remove definition: `tempfile`

(F811)